rackcity/views/user_views.py

```diff
@@ -450,6 +450,25 @@ def user_get_groups(request):
             },
             status=HTTPStatus.BAD_REQUEST,
         )
+    permissions = get_permissions(user)
+    return JsonResponse(
+        permissions,
+        status=HTTPStatus.OK,
+    )
+
+
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def user_get_my_groups(request):
+    user = request.user
+    permissions = get_permissions(user)
+    return JsonResponse(
+        permissions,
+        status=HTTPStatus.OK,
+    )
+
+
+def get_permissions(user):
```

Comment:
Git is showing this change weird but I factored this out and added an endpoint for getting your own permissions

Reply:
i like